ui/components-react/src/components-react/filter-builder/FilterBuilderRule.tsx

```diff
@@ -72,9 +73,11 @@ export function PropertyFilterBuilderRuleRenderer(props: PropertyFilterBuilderRu
 
   return <div className="rule">
     <div className="rule-remove-action">
-      <IconButton onClick={removeRule} styleType="borderless" size="small">
-        <SvgDelete />
-      </IconButton>
+      {property || !isRemovable
```

Comment:
`!isRemoveable` looks incorrect. Remove button will be shown when `isRemoveable` is set to `false`.

Reply:
fixed